docs/api/README.md

```diff
@@ -263,6 +265,57 @@ const takeLatest = (patternOrChannel, saga, ...args) => fork(function*() {
 
 You can also pass in a channel as argument and the behaviour is the same as [takeLatest(pattern, saga, ...args)](#takelatestpattern-saga-args).
 
+### `takeLeading(pattern, saga, ...args)`
+
+Spawns a `saga` on each action dispatched to the Store that matches `pattern`. And automatically ignores
```

Comment:
> And automatically ignores any coming new `saga` task if it's still running.

This is imho slightly inaccurate. It would be better to just say that it blocks until spawned saga completes AND then starts to listen for a pattern again.